helm/idol-nifi/templates/nifi/nifi.yaml

```diff
@@ -0,0 +1,204 @@
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: nifi
+  labels: {{- include "idol-nifi.labels" . | nindent 4 -}}
+    name: nifi
+    app: nifi
+  annotations:
+    app.kubernetes.io/name: nifi
+    app.kubernetes.io/part-of: nifi
+spec:
+  revisionHistoryLimit: 1
+  serviceName: nifi
+  podManagementPolicy: OrderedReady
+  selector:
+    matchLabels:
+      app: nifi
+  template:
+    metadata:
+      labels: {{- include "idol-nifi.labels" . | nindent 8 -}}
+        app: nifi
+    spec:
+      terminationGracePeriodSeconds: 90
+      automountServiceAccountToken: false
+      enableServiceLinks: false
+      setHostnameAsFQDN: true
+      dnsPolicy: ClusterFirstWithHostNet
+      restartPolicy: Always
+      securityContext:
+        runAsGroup: 1000
+        runAsUser: 1000
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+      initContainers:
+        - name: wait-for-zookeeper
+          image: docker.io/busybox:1.36
+          command:
+          - sh
+          - -c
+          - |
+            echo "Connecting to Zookeeper ${NIFI_ZK_CONNECT_STRING}"
+            until nc -vzw 1 zookeeper 2181 ; do
+              echo "Waiting for zookeeper to start"
+              sleep 3
+            done
+          envFrom:
+          - configMapRef:
+              name: nifi-cm
+              optional: false
+          resources:
+            requests:
+              cpu: 20m
+              memory: 10Mi
+            limits:
+              cpu: 20m
+              memory: 10Mi
+          securityContext:
+            allowPrivilegeEscalation: false
+            privileged: false
+            readOnlyRootFilesystem: true
+            capabilities:
+              drop:
+                - ALL
+      imagePullSecrets:
+        {{- range .Values.imagePullSecrets }}
+        - name: {{ . }}
+        {{- end }}
+      containers:
+      - name: nifi
+        image: {{ .Values.idolImageRegistry }}/{{ .Values.nifi.imageName }}:{{ .Values.idolVersion }}
+        #image: idol-docker-images.svsartifactory.swinfra.net/nifi-full:23.4
```

Comment:
Intended to commit this line?

Reply:
[bdcf610](https://github.com/opentext-idol/idol-containers-toolkit/pull/12/commits/bdcf6102a5b4a6abe48f8d6a95c252e1e6a5c6c4)